disconnect-entitylist.json

```diff
@@ -2354,11 +2354,21 @@
     "Complex Media": {
         "properties": [
             "complexmedianetwork.com",
-            "complex.com"
+            "complex.com",
+            "collider.com",
+            "solecollector.com",
+            "pigeonsandplanes.com",
+            "theridechannel.com",
+            "firstwefeast.com"
         ],
         "resources": [
             "complexmedianetwork.com",
-            "complex.com"
+            "complex.com",
+            "collider.com",
```

Comment:
As per [bug 1315740](https://bugzilla.mozilla.org/show_bug.cgi?id=1315740) these are top-level properties, not resources that other sites pull in. Also, they're not on the Disconnect blocklist, so they only need to be in the `properties` section, not under `resources`.